conda_forge_tick/update_upstream_versions.py

```diff
@@ -72,6 +72,8 @@ def get_version(self, url):
                     ver = ver[len(prefix) :]
             if any(s in ver for s in self.dev_vers):
                 continue
+            # Extract vesion number starting at the first digit.
+            ver = re.search(r"(\d+.*)", ver).group(0)
```

Comment:
```suggestion
            ver = re.search(r"(\d+[^\s]*)", ver).group(0)
```

might be a tiny bit safer?